docs/content/preview/api/ysql/the-sql-language/statements/streaming_start_replication.md

```diff
@@ -38,18 +38,36 @@ The log sequence number from where to start the streaming from.
 
 The name of an option passed to the slot's logical decoding plugin.
 
+The applicable options accepted by the command depends on the output plugin of the replication slot. They can be viewed in the respective documentation of the output plugin itself.
+
+For `pgoutput` and `yboutput`, check the section [53.5.1. Logical Streaming Replication Parameters](https://www.postgresql.org/docs/11/protocol-logical-replication.html) in the PG documentation.
```

Comment:
Do we need to document somehow `yboutput`? Or say how/if it differes from `pgoutput` ?

Reply:
Yes but not in the syntax section. cc: @asrinivasanyb